src/Namespaces/TicketMessagesNamespace.php

```diff
@@ -20,15 +20,17 @@ class TicketMessagesNamespace extends AbstractNamespace
 	 * @param int    $ticketId
 	 * @param string $text
 	 * @param bool   $interimNotice
+     * @param array $claimMessageFiles
```

Comment:
```suggestion
     * @param Hitmeister\\Component\\Api\\Transfers\\MessageClaimFileTransfer[] $claimMessageFiles
```

Reply:
Also, the `Hitmeister\\Component\\Api\\Transfers\\MessageClaimFileTransfer` transfer itself should be added to the repo

Reply:
Should be there now